components/SignUpServicesForm/SignUpServicesFormWithModal.tsx

```diff
@@ -7,23 +7,29 @@ import { useState } from 'react'
 import { useTranslation } from 'react-i18next'
 import Dialog from '../Dialog'
 
-
 export const SignUpServicesFormWithModal = () => {
   const { t } = useTranslation()
 
   const [showDialog, setShowDialog] = useState(false)
-  const [showForm, setShowForm] = useState<'transportGoods' | 'transportPersons'>();
+  const [showForm, setShowForm] = useState<
+    'transportGoods' | 'transportPersons'
+  >()
 
   const onTransportGoodsSubmit = (data: TransportServicesRequest) => {
-    console.log(data);
-    setShowDialog(false);
-    setShowForm(undefined);
+    console.log(data)
+    setShowDialog(false)
+    setShowForm(undefined)
   }
 
   const onTransporPersonsSubmit = (data: TransportServicesRequest) => {
-    console.log(data);
-    setShowDialog(false);
-    setShowForm(undefined);
+    console.log(data)
+    setShowDialog(false)
```

Comment:
`handleDialogDismiss` could be called here for these two states. since the submit action also dismisses the dialog